engine/storage/snapshot/src/main/java/org/apache/cloudstack/storage/snapshot/StorageSystemSnapshotStrategy.java

```diff
@@ -484,7 +483,7 @@ else if (volumeInfo.getFormat() == ImageFormat.OVA || volumeInfo.getFormat() ==
         }
 
         SnapshotResult result = null;
-        SnapshotInfo snapshotOnPrimary = null;
+        SnapshotInfo snapshotOnPrimary;
```

Comment:
I guess the removal of this may create NPE.

Reply:
how is this, @nitin-maharana?

Reply:
This is an IntelliJ-recommended change. The IDE pointed out (correctly) that we did not need to assign null on this line because snapshotOnPrimary is later assigned a value regardless of the path the code takes.

Reply:
@DaanHoogland, My bad, wrong interpretation!! As @mike-tutkowski explained, that's redundant.  Thanks!!